### ISSUE TITLE
target training

### ISSUE THREAD

**Issue by j-harshith-reddy** [Jason-cse/ML-wine_quality#29]
data['quality_class'] = (data['quality'] >= 6).astype(int)
print(data['quality_class'].value_counts(normalize=True).rename('proportion'))

X = data.drop(columns=['quality', 'quality_class'])
y = data['quality_class']

numeric_features = X.select_dtypes(include=np.number).columns.tolist()
categorical_features = X.select_dtypes(exclude=np.number).columns.tolist()
print('Numeric features:', len(numeric_features))
print('Categorical features:', categorical_features)

X_train, X_test, y_train, y_test = train_test_split(
    X, y, test_size=0.2, stratify=y, random_state=RANDOM_STATE
)
print('Train shape:', X_train.shape, ' Test shape:', X_test.shape)
